src/libsync/connectionvalidator.cpp

```diff
@@ -144,7 +144,12 @@ void ConnectionValidator::slotStatusFound(const QUrl&url, const QVariantMap &inf
 // status.php could not be loaded (network or server issue!).
 void ConnectionValidator::slotNoStatusFound(QNetworkReply *reply)
 {
-    qDebug() << Q_FUNC_INFO << reply->error() << reply->errorString();
+    qDebug() << Q_FUNC_INFO << reply->error() << reply->errorString() << reply->peek(1024);
```

Comment:
Yes, maybe it's better if this is generally needed, so we'd always load the credentials at the start from keychain.
@ogoffart @dragotin @danimo Any input?